thetis/options.py

```diff
@@ -110,7 +123,10 @@ class PressureProjectionTimestepperOptions2d(TimeStepperOptions):
 
 
 class ExplicitTimestepperOptions2d(ExplicitTimestepperOptions):
-    """Options for 2d explicit time integrator"""
+    """
```

Comment:
should we rename it `ExplicitSWETimestepperOptions2d` then? And similar for cases below? Also given my next comment on inheritance, I think it's clearer to explain what the relation between the two are.